### PR TITLE
Create a docker container

### PR DESCRIPTION
**Pull request by rjzondervan**

### CODE REVIEW

File: .github/workflows/dockerimage.yml [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+          echo "APP_NAME=$NAME" >> $GITHUB_ENV
+      - name: Print app name
+        run: echo "APP_NAME = $APP_NAME"
+      - name: Setting APP_ENV to dev
+        run: |
+          echo "APP_ENV=dev">> $GITHUB_ENV
+          echo "APP_BUILD=dev">> $GITHUB_ENV
+          echo "set APP_ENV to $APP_ENV"
+      - name: Setting APP_ENV to prod
+        if: contains( github.ref, 'master' ) || contains( github.base_ref, 'master' )
```
Comment (WilcoLouwerse) [inline .github/workflows/dockerimage.yml:39]: Shouldn't this be main now ? or at least not master any more

File: .github/workflows/dockerimage.yml [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        run: |
+          echo "APP_ENV=latest">> $GITHUB_ENV
+          echo "APP_BUILD=latest">> $GITHUB_ENV
+          echo "set APP_ENV to $APP_ENV"
+      - name: Setting APP_ENV to stag
+        if: contains( github.ref, 'staging' ) || contains( github.base_ref, 'staging' )
```
Comment (WilcoLouwerse) [inline .github/workflows/dockerimage.yml:45]: Shouldn't staging be removed?

File: .github/workflows/dockerimage.yml [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+      - name: Login to Container Registry
+        id: containerregistry-login
+        run: |
+         if [ "${{ secrets.GITHUB_TOKEN }}" != "" ]; then
+           echo ${{ secrets.GITHUB_TOKEN }} | docker login ghcr.io -u $(cut -d'/' -f1 <<< $GITHUB_REPOSITORY) --password-stdin
+           echo "##[set-output name=success;]true"
+         else
+           echo "##[set-output name=success;]false"
+         fi
+      - if: steps.containerregistry-login.outputs.success == 'true' && (github.ref == 'refs/heads/master' || github.ref == 'refs/heads/staging' || github.ref == 'refs/heads/development' || github.ref == 'refs/heads/refinement-demo')
```
Comment (WilcoLouwerse) [inline .github/workflows/dockerimage.yml:73]: master and staging here, shouldn't this be (only) main

File: .env [hunk 4]
```diff
@@ -33,7 +33,7 @@ APP_TYPE=component
 # The urls/ip's and hosts on wich this api is available
 
 TRUSTED_PROXIES=10.0.0.0/8,172.16.0.0/12,192.168.0.0/16
-TRUSTED_HOSTS=^.+$
+TRUSTED_HOSTS=
```
Comment (WilcoLouwerse) [inline .env:36]: What is the difference between these 2 ^ ?

**Review by WilcoLouwerse — CHANGES_REQUESTED** [review]

File: .github/workflows/dockerimage.yml (hunk 1 above)
Comment (rjzondervan) [reply .github/workflows/dockerimage.yml:39]: Good point

File: .env (hunk 4 above)
Comment (rjzondervan) [reply .env:36]: I'm not sure, they are not used as far as I know, but Docker cannot build the first one

**Review by WilcoLouwerse — APPROVED** [review]

**Pull request merged**